src/main/resources/org/sonatype/nexus/ci/nxrm/v3/CreateTagBuilder/config.groovy

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2016-present Sonatype, Inc. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package org.sonatype.nexus.ci.nxrm.v3.pipeline.CreateTagStep
+
+import org.sonatype.nexus.ci.util.NxrmUtil
+
+def f = namespace(lib.FormTagLib)
+def l = namespace(lib.LayoutTagLib)
+
+l.css(src: "${rootURL}/plugin/nexus-jenkins-plugin/css/nexus.css")
```

Comment:
When running with `mvn hpi:run` the browser dev mode consistently shows that this css file cannot be found. Is there a reason for that?

Reply:
Again a copy paste. Will double check it's validity.